### DISCUSSION TITLE
Recommendation regarding mail flood to golangci/team

### DISCUSSION THREAD

**Discussion by breml** [golangci/golangci-lint#2930] [General] [answered]
Since I contributed a linter to golangci-lint, I have been invited to the golangci/team, an invitation I happily accepted, because I thought it would be good to be further involved and informed, if there are issues related to the linter I contributed. But now, I get annoyed by the amount of email notifications I get because of dependabot and the following interactions on the respective PR.

As a first reaction, I decided to set my Github watch setting for all repos other than golangci/golangci-lint to ignoring. But this is not the setting I prefer to use for the golangci-lint repo, because as mentioned, I would like to get informed, if there is something wrong with the linters I wrote.

I would like to open the discussion, if the settings for dependabot can be changed, such that a smaller group of maintainers get informed, if there are dependencies to be updated. At the moment, each of the dependabot PR is assigned to golangci/team, which includes more than 100 people.

**Comment by bombsimon** [comment, marked as answer]
Do you still get notifications if you leave the team that gets assigned or ignore notifications? I don't see the reason to be a member of `golangci/team` if you only want to watch issues or PRs related to a single linter so that might be an alternative. Usually we ping authors or file upstream issues for specific linters so you don't really have to even be a member of the org to get notified.

**Reply by ldez** [reply]
https://github.com/golangci/golangci-lint/pull/2932

**Reply by breml** [reply]
@ldez Thanks for the PR, I guess, this will resolve my issue.
